index.html

```diff
@@ -10,185 +10,225 @@
+                    </p>
+                    <ul>
+                        <li id="html"><a href="#"> html</a></li>
+                        <li id="css"><a href="#">css</a></li>
+                        <li id="javaScript"><a href="#">javaScript</a></li>
+                    </ul>
+                    <div class="to-project"><a href="#">See Project</a></div>
+                </div>
+            </div>
         </div>
-        <div class="container container4">
-            <div class="Card1"><img src="images/img4.png" alt="" class="card1"></div>
-            <div>
-                <h1>Multi-Post <br>Stories</h1>
+
+        <section class="About-section">
+            <div class="order-1">
+                <div class="about-title">About <br> Myself</div>
+                <p class="para2">Hello I'm a software developer I can help you build a product, feature or website
+                    look through some of my work and experience! if you like what you see and havd a Project
+                    you need coded, don't hesitate to contact me.
+                    <div class="connect">LET'S CONNECT</div>
+                </p>
+                <ul>
+                    <li><a href=""><img src="./images/github.png" /></a></li>
+                    <li><a href=""><img src="./images/linkedin.jpeg" /></a></li>
+                    <li><a href=""><img src="./images/sign.png" /></a></li>
+                    <li><a href=""><img src="./images/twiter.png" /></a></li>
+                    <li><a href=""> <img src="./images/Muliti.png" /></a></li>
+                </ul>
+
+                <div class="get">Get my resume</div>
             </div>
-            <div class="lister">
-                <p><strong>CANOPY</strong></p>
-                <div class="point"></div>
-                <P>Back End Dev</P>
-                <div class="point"></div>
-                <p>2015</p>
+            <div class="order-2">
+                <ul>
+                    <li>
+                        <div class="lang-dropper">Languages <img src="./images/dropBtn.png" id="dropBtn1" alt="">
+                        </div>
+                    </li>
+                </ul>
+
+                <div class="lang-flex">
+                    <div class="lang"><img src="images/js.png" alt="">javaScript</div>
+                    <div class="lang"><img src="images/html.png" alt="">HTML</div>
+                    <div class="lang"><img src="images/CSS.png" alt="">CSS</div>
+                </div>
+
+                <ul class="footer-list">
+                    <li>
+                        <div class="frame">Frameworks <img src="./images/dropBtn2.png" id="dropBtn2" alt=""></div>
+                    </li>
+                    <li>
+                        <div class="frame"><span id="span">Skills</span><img src="./images/dropBtn2.png" id="dropBtn2"
+                                alt=""></div>
+                    </li>
+                </ul>
             </div>
```

Comment:
Kindly split the about section into two blocks the right one is `40%` and the left one `60%`
![image](https://user-images.githubusercontent.com/24703171/155525204-8f92c006-7086-4074-8348-733c1218adca.png)